sway-lib-std/src/u256.sw

```diff
@@ -293,28 +293,26 @@ impl U256 {
 impl core::ops::Ord for U256 {
     #[allow(deprecated)]
     fn gt(self, other: Self) -> bool {
-        self.a > other.a ||
-                (self.a == other.a && (self.b > other.b ||
-                                      (self.b == other.b && (self.c > other.c ||
-                                                            (self.c == other.c && self.d > other.d)))) )
-        }
+        self.a > other.a || (self.a == other.a && (self.b > other.b || (self.b == other.b && (self.c > other.c || (self.c == other.c && self.d > other.d)))))
+    }
 
     #[allow(deprecated)]
     fn lt(self, other: Self) -> bool {
-        self.a < other.a ||
-                (self.a == other.a && (self.b < other.b ||
-                                      (self.b == other.b && (self.c < other.c ||
-                                                            (self.c == other.c && self.d < other.d)))) )
+        self.a < other.a || (self.a == other.a && (self.b < other.b || (self.b == other.b && (self.c < other.c || (self.c == other.c && self.d < other.d)))))
     }
 }
 
 #[test]
 fn test_u256_ord() {
     assert(U256::from((0, 0, 0, 1)) < U256::from((0, u64::max(), 0, 0)));
-    assert(!(U256::from((0, 0, 0, 1)) > U256::from((0, u64::max(), 0, 0))));
+    assert(
```

Comment:
It appears our default maximum line length is much shorter than rustfmt's. I can add 22 characters to this line before rustfmt will wrap it. Perhaps we should use the same as rustfmt.